src/app/[locale]/layout.tsx

```diff
@@ -12,7 +12,7 @@ type Props = {
   params: { locale: string };
 };
 export async function generateStaticParams() {
-  return ['it', 'en'].map((locale) => ({ locale }));
+  return ['it'].map((locale) => ({ locale }));
```

Comment:
We dont have the translation keys for English. When you run next build it generates static pages without translations for english and gives warnings in console beacuse it can not find the keys to translate. Should be adedd back once the English translation file is finished.